manifests/infra/prometheus/overlays/development/kustomization.yaml

```
@@ -2,3 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
 - ../../base
+- https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.52.1/bundle.yaml
```

```suggestion
resources:
- https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.52.1/bundle.yaml
- ../../base
```
今知ったのですが、`bases`ではなく`resources`を使うのがよいみたいです。
> The bases field was deprecated in v2.1.0

https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/bases/

> Resources will be read and processed in depth-first order.

https://kubectl.docs.kubernetes.io/references/kustomize/resource/
を正しく解釈できているか自信ありませんが、`bundle.yaml`で定義しているリソースに他は依存しているため、順番を入れ替えておきます。
更新時は問題なく動きそうですが、ローカル環境など新規に構築する場合に動くか疑問です。

---

勉強になりました！

---

Good catch! @oke-py